controllers/restaurants/restaurantsController.go

```diff
@@ -53,3 +53,22 @@ func Create(c *gin.Context) {
 		})*/
 
 }
+
+func Update(c *gin.Context) {
+	var rest models.Restaurant
+
+	err := c.ShouldBindJSON(&rest)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
à la place du panic tu peux mettre
l
```suggestion
	if err != nil {
		log.Println(err)
		c.JSON(http.StatusBadRequest, gin.H{
			"error": err.Error(),
		})
	}
```